qtconsole/manager.py

```diff
@@ -33,6 +33,10 @@ class QtKernelManager(KernelManager, QtKernelManagerMixin):
     client_class = DottedObjectName('qtconsole.client.QtKernelClient')
     autorestart = Bool(True, config=True)
 
+    def __init__(self, *args, **kwargs):
+        super(QtKernelManager, self).__init__(*args, **kwargs)
```

Comment:
```suggestion
        super().__init__(*args, **kwargs)
```

Our next version is going to be compatible only with Python 3.